xbmc/platform/win32/XTimeUtils.cpp

```diff
@@ -34,27 +32,30 @@ static_assert(offsetof(SystemTime, day) == offsetof(SYSTEMTIME, wDay));
 static_assert(offsetof(SystemTime, hour) == offsetof(SYSTEMTIME, wHour));
 static_assert(offsetof(SystemTime, minute) == offsetof(SYSTEMTIME, wMinute));
 static_assert(offsetof(SystemTime, second) == offsetof(SYSTEMTIME, wSecond));
+
+int64_t FileTimeToInt64(const FILETIME& ft)
+{
+  return static_cast<int64_t>((uint64_t(ft.dwHighDateTime) << 32) | ft.dwLowDateTime);
+}
+
 } // namespace
 
-uint32_t GetTimeZoneInformation(TimeZoneInformation* timeZoneInformation)
+std::tuple<bool, int64_t> GetTimezoneBias(const SystemTime& time)
 {
-  if (!timeZoneInformation)
-    return KODI_TIME_ZONE_ID_INVALID;
-
-  TIME_ZONE_INFORMATION tzi{};
-  uint32_t result = ::GetTimeZoneInformation(&tzi);
-  if (result == TIME_ZONE_ID_INVALID)
-    return KODI_TIME_ZONE_ID_INVALID;
-
-  timeZoneInformation->bias = tzi.Bias;
-  timeZoneInformation->daylightBias = tzi.DaylightBias;
-  timeZoneInformation->daylightDate = *reinterpret_cast<const SystemTime*>(&tzi.DaylightDate);
-  timeZoneInformation->daylightName = FromW(tzi.DaylightName);
-  timeZoneInformation->standardBias = tzi.StandardBias;
-  timeZoneInformation->standardDate = *reinterpret_cast<const SystemTime*>(&tzi.StandardDate);
-  timeZoneInformation->standardName = FromW(tzi.StandardName);
-
-  return result;
+  // Compute offset = local_filetime - utc_filetime
+  FILETIME utcFt{};
+  if (!::SystemTimeToFileTime(reinterpret_cast<const SYSTEMTIME*>(&time), &utcFt))
```

Comment:
```suggestion
  if (FALSE == ::SystemTimeToFileTime(reinterpret_cast<const SYSTEMTIME*>(&time), &utcFt))
```

`SystemTimeToFileTime` retuns `BOOL` type (which is `int`)